starky/src/stark_verifier_circom.rs

```diff
@@ -1589,24 +1589,292 @@ template StarkVerifier() {{
         }}
     }}
 }}
+
 "#,
         nLastBits,
         1 << nLastBits,
         1 << maxDegBits,
         1 << nLastBits
     ));
 
+    ///////
+    // Aggregation Stage
+    ///////
+
+    if options.agg_stage {
+        res.push_str(&format!(
+            r#"
+template Recursive2() {{
```

Comment:
This is actually recursivef?

Reply:
It is recursive2 because it combines two starkVerifier() into one circuit.

Reply:
[here ](https://github.com/0xEigenLabs/eigen-zkvm/pull/71/files#diff-07e89d24351a3dc427f8f460d0e901d0426d393c388b4e4fc7d5d569982a4dd8R82) generates recursivef